packages/kotti-ui/source/kotti-field-file-upload/components/DropArea.vue

```diff
@@ -0,0 +1,276 @@
+<template>
+	<div
+		ref="dropAreaRef"
+		:class="classes"
+		:data-test="dataTest ? `${dataTest}.dropArea` : undefined"
+		:tabIndex="_tabIndex"
+		@click.prevent="onClick"
+		@dragleave.prevent.stop="isDragging = false"
+		@dragover.prevent.stop="isDragging = true"
+		@drop.prevent.stop="onDrop"
+		@mouseenter.prevent.stop="isHovering = true"
+		@mouseleave.prevent.stop="isHovering = false"
+	>
+		<div class="kt-field-file-upload__drop-area__description">
+			<i class="yoco" v-text="_icon" />
+			<a v-text="translations.text.clickToUpload" />
+			{{ translations.text.dragAndDrop }}
+		</div>
+		<div
+			v-if="showInformation"
+			class="kt-field-file-upload__drop-area__information"
+		>
+			{{ informationText }}
+			<a
+				v-if="infoUrl"
+				:href="infoUrl"
+				rel="noopener noreferrer"
+				target="_blank"
+				@click.stop
+				v-text="translations.text.learnMore"
+			/>
+		</div>
+		<input
+			v-show="false"
```

Comment:
:) out of curiosity

did you run into issues with the css display none along with using a reference?

Reply:
Just tested, no issues, it's working. 
I just did it this way, will change it.

Reply:
you don't need to change it, I was curious if it'd caused it - but I don't see any reason why it shouldn't be a v-show